gradle/libs.versions.toml

```diff
@@ -0,0 +1,26 @@
+[versions]
+kotlin = "1.6.0"
+extensions = "1.0.1-pre.264-kotlin-1.5.31"
```

Comment:
Advantage of this versions file is it can be structured to have versions that should match near each other.
In this case, the `extensions` is behind, and should be upgraded. I'll upgrade in a subsequent PR.